backend/db/config.py

```diff
@@ -93,8 +95,12 @@ def get_pg_connect_url(local=False):
 ]
 # STANDALONE_TABLES: Not derived from any other table, nor used to derive any other table/view. Used for QC testing.
 STANDALONE_TABLES = [
+    'concept_set_json',
     'junk',
     'relationship',
+    'session_concept',
+    'sessions',
+    'test',
```

Comment:
New standalone tables in `n3c`:
    'concept_set_json',
    'session_concept',
    'sessions',
    'test',